src/app/job-registration-form/job-registration-form.component.ts

```diff
@@ -46,7 +46,7 @@ export class JobRegistrationFormComponent implements OnInit {
             email: ['', [Validators.email, Validators.maxLength(50)]],
             contactName: ['', [Validators.required, Validators.maxLength(64), this.noWhiteSpaceValidator]],
             website: ['', [Validators.maxLength(64)]],
-            phone: ['', [Validators.required]],
+            phone: ['', [Validators.required, Validators.max(999999999999999999)]],
```

Comment:
Why this number? What's the point of this validator?

Reply:
Max length validator wasn't working on the phone field for some reason, so i added the max number value that we felt was needed for a phone number as otherwise there could be an infinite length of phone number